rosetta/services/construction.go

```diff
@@ -0,0 +1,89 @@
+package services
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/coinbase/rosetta-sdk-go/server"
+	"github.com/coinbase/rosetta-sdk-go/types"
+	ethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/harmony-one/harmony/common/denominations"
+	hmyTypes "github.com/harmony-one/harmony/core/types"
+	"github.com/harmony-one/harmony/hmy"
+	"github.com/harmony-one/harmony/rosetta/common"
+	stakingTypes "github.com/harmony-one/harmony/staking/types"
+)
+
+const (
+	// DefaultGasPrice ..
+	DefaultGasPrice = denominations.Nano
```

Comment:
My thought here was that a consumer can import our rosetta package and have access to what the implementation uses as the default gas price. This is needed since a user can provide the `GasPriceMultiplier` when constructing a transaction, and this number is multiplied by the `DefaultGasPrice` for the final gas price of the transaction.